docs/hub/spaces-zerogpu.md

```diff
@@ -81,6 +96,29 @@ def generate(prompt):
 
 This sets the maximum function runtime to 120 seconds. Specifying shorter durations for quicker functions will improve queue priority for Space visitors.
 
+### Dynamic duration
+
+`@spaces.GPU` also supports dynamic durations.
+
+Instead of directly passing a duration, simply pass a callable that takes the same inputs as your decorated function and returns a duration:
```

Comment:
```suggestion
Instead of directly passing a duration, simply pass a callable that takes the same inputs as your decorated function and returns a duration value:
```